composer.json

```diff
@@ -2,43 +2,34 @@
     "name": "laravel/laravel",
     "type": "project",
     "description": "The Laravel Framework.",
-    "keywords": ["framework", "laravel"],
+    "keywords": [
+        "framework",
+        "laravel"
+    ],
     "license": "MIT",
     "require": {
         "php": "^8.2",
-        "archtechx/enums": "^0.3.1",
```

Comment:
This package was only used for the `Lowercase()` rule, which is now built-in Laravel.